OrchardCore.Commerce/Views/TextProductAttributeValue_Hidden.cshtml

```diff
@@ -4,12 +4,12 @@
     var attr = Model.AttributeValue as TextProductAttributeValue;
     var name = $"{Model.NamePrefix ?? "line"}.Attributes[{index}]";
     var form = string.IsNullOrEmpty(Model.Form) ? "" : $" form=\"{Model.Form}\"";
-    if (attr.Value != null)
+    if (attr?.Value != null)
```

Comment:
In the `_Hidden` views of the other two attribute types we don't check `attr` for `null`. Let's make this consistent. Also, I'm thinking that if we expect `attr` to be of a given type, then lets use direct casts everywhere, shall we?